src/main/java/net/imagej/ops/descriptor3d/AbstractPolygon.java

```diff
@@ -0,0 +1,136 @@
+package net.imagej.ops.descriptor3d;
+
+import java.util.List;
+
+/**
+ * An {@link AbstractPolygon} consists of vertices and neighbors.
+ * 
+ * @author Tim-Oliver Buchholz, University of Konstanz.
+ *
+ */
+public abstract class AbstractPolygon {
```

Comment:
can we give this class another name as there already exists a `Polygon` in imglib2-roi. Maybe `UpdateablePointSet`? Because this is not really a `Polygon`. Furthermore, I would prefer package visibility in for this class.